blocks/core/ff_module/ff_module-filter/ff_module-filter.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+$(function(){ 
+	$("[data-ff-action='filter-control']").click(function(){
+		var filterDropdown = $(this),
+		      filterIcon = filterDropdown.find("[data-ff-icon='filter-dropdown-icon']"),
+		      filterContent = filterDropdown.closest("[data-ff='filter']").find("[data-ff='filter-content']");
```

Comment:
Probably not, it's just my lack of `js` skills. I want to find the `content` in a reliable way, so thought it best to transvers up the tree to a known node, then `find()` within that.